modules/microshift-preparing-for-image-building-bootc.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-about-rhel-image-mode
+
+:_mod-docs-content-type: CONCEPT
+[id="preparing-for-image-building_{context}"]
+= Preparing for image building
+
+Use the image builder tool to compose customized {microshift-short} bootc images optimized for edge deployments. You can run a {microshift-short} cluster with your applications on a {op-system-image} virtual machine for development and testing first, then use your whole solution in edge production environments.
+
+Use the following {op-system-base} documentation to understand the full details of using {op-system-image}:
+
+* To build an {op-system-base} {op-system-version} image for a given CPU architecture, you need a {op-system-base} {op-system-version} build host of the same CPU architecture that meets the link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/setting-up-image-builder_composing-installing-managing-rhel-for-edge-images#edge-image-builder-system-requirements_setting-up-image-builder[Image builder system requirements].
+
+* Follow the instructions in link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/setting-up-image-builder_composing-installing-managing-rhel-for-edge-images#edge-installing-image-builder_setting-up-image-builder[Installing image builder] to install image builder and the `composer-cli` tool.
```

Comment:
These are wrong links for bootc / image mode. The links point to composer-cli, which is an "old" tool used for building ostree images - not bootc.